src/api/controllers/user/add-recipe.js

```diff
@@ -0,0 +1,43 @@
+import { getUserById } from './helpers.js';
+import { createRecipeToDb } from './helpers.js';
+import Jimp from 'jimp';
+import { nanoid } from 'nanoid';
+
+async function addRecipe(req, res, next) {
+  try {
+    const id = req.user.id;
+    const recipe = JSON.parse(req.body.recipe);
+    const user = await getUserById(id);
+    if (!user) {
+      return res.status(401).json({ message: 'Nope' });
+    }
+    if (!recipe) {
+      return res.status(401).json({ message: 'Nope' });
```

Comment:
🤣

Reply:
I know. It's just a base thing. :D

Reply:
Please comment things that need to be changed next time 🤞